lib/pages/connect/connect_page_mixin.dart

```diff
@@ -37,13 +37,19 @@ mixin ConnectPageMixin {
   bool supportsSso(BuildContext context) =>
       (PlatformInfos.isMobile ||
           PlatformInfos.isWeb ||
-          PlatformInfos.isMacOS) &&
+          PlatformInfos.isMacOS ||
+          PlatformInfos.isLinux) &&
       supportsFlow(context: context, flowType: 'm.login.sso');
 
   bool supportsLogin(BuildContext context) =>
       supportsFlow(context: context, flowType: 'm.login.password');
 
   String _getRedirectUrlScheme(String redirectUrl) {
+    // Remove when package limitation will be fixed
+    // https://pub.dev/packages/flutter_web_auth_2#windows-and-linux
+    if (PlatformInfos.isLinux || PlatformInfos.isWindows) {
+      return "http://localhost:60665";
```

Comment:
can you explain a bit?

Reply:
This is the URL which is observed by the app on those platforms to know if SSO login succeed or not.
It should be the same in `redirectUrl` and `callbackUrlScheme` . That's where the problem was